cypress/support/fragments/eholdings/eHoldingsPackages.js

```diff
@@ -81,7 +81,35 @@ export default {
   verifyListOfExistingPackagesIsDisplayed: () => {
     cy.expect(resultSection.find(List()).exists());
   },
+  sortPackagesByTitlesCount({ minTitlesCount = 100 } = {}) {
```

Comment:
Why don't use  List and ListItem interactors instead of  bare cypress

Reply:
Fixed